examples/fill-in-middle/main.py

```diff
@@ -1,22 +1,9 @@
 from ollama import generate
 
-prefix = '''def remove_non_ascii(s: str) -> str:
-    """ '''
-
-suffix = """
-    return result
-"""
-
-
 response = generate(
-  model='codellama:7b-code',
-  prompt=f'<PRE> {prefix} <SUF>{suffix} <MID>',
```

Comment:
This example should remain the same except use prompt and suffix instead of templating prompt with f-string